traits_enaml/testing/tests/test_enaml_test_assistant.py

```diff
@@ -69,17 +80,13 @@ def test_find_all_enaml_widgets(self):
 
 
 class TestEnamlTestHelperFunctions(EnamlTestAssistant, unittest.TestCase):
-
     def test_print_enaml_widget_tree(self):
         view, _ = self.parse_and_create(ENAML_SOURCE)
-        stream = StringIO.StringIO()
-        old_stdout = sys.stdout
-        try:
-            sys.stdout = stream
+        stream = io.StringIO()
+        with redirect_stdout(stream):
```

Comment:
I think `with redirect_stdout(stream), contextlib.closing(stream):` will work here if you want to avoid the explicit stream close at the end.

Reply:
i forget what the order of context manager enter/exit is when context managers are chained but it doesn't really matter here.

Reply:
@jwiggins refactored. Take another look?